state/runtime/evm/instructions_test.go

```diff
@@ -437,6 +437,7 @@ func TestCreate(t *testing.T) {
 	}
 }
 
+// EIP2929: check some test return error when triggered from base
```

Comment:
Could you add some unit tests that would test different gas consumptions per opcodes? I see that you have opted to test it out on a higher level of abstraction (namely on the EVM level), but I think it would be beneficial for us to have some "atomic" unit tests (namely the ones that would test gas consumption per specific op code, depending on which hard fork is enabled).
WDYT?

Reply:
yeah that makes sense to me, I will add those unit test.

For these EVM level transactions I have also verified the cost of individual opcodes by printing call stack.

Reply:
added uint test cases for some opcodes and will add more with eip-2930 implementation